src/main/resources/org/semux/api/swagger/v2.3.0.json

```diff
@@ -2434,6 +2434,56 @@
                 ]
             }
         },
+        "/transaction/call-local": {
```

Comment:
Suggest using `/call` as the endpoint to distinguish message calls from transactions.

Also, suggest removing the `local` parameter from the `/transaction/call`.